caffe2/python/convnet_benchmarks.py

```diff
@@ -563,45 +565,81 @@ def Inception(order, cudnn_ws, model_path=""):
     model.net.AveragedLoss(xent, "loss")
     return model, 224
 
-def Resnet50(order, cudnn_ws, model_path=""):
-    if model_path == "":
-        print("ERROR: please specify paths to init_net and predict_net protobufs for Resnet50")
-        exit(1)
-    device_opts = caffe2_pb2.DeviceOption()
-    device_opts.device_type = caffe2_pb2.HIP
-    device_opts.hip_gpu_id = 0
 
-    INIT_NET_PB = os.path.join(model_path, "init_net.pb")
-    PREDICT_NET_PB = os.path.join(model_path, "predict_net.pb")
-    init_def = caffe2_pb2.NetDef()
-    with open(INIT_NET_PB, 'rb') as f:
-        init_def.ParseFromString(f.read())
-        init_def.device_option.CopyFrom(device_opts)
+def Resnet50(args):
```

Comment:
Could just use the signature Resnet50(order, cudnn_ws, model_path="") just like the other model creator functions

Reply:
Resnet needs other args as well, like args.forward_only, args.batch_size which necessitates a different signature.